fem/bilinearform.cpp

```diff
@@ -421,23 +422,30 @@ void BilinearForm::Assemble(int skip_zeros)
                         "invalid element marker for domain integrator #"
                         << k << ", counting from zero");
          }
+
+         if (domain_integs[k]->Patchwise())
+         {
+            MFEM_VERIFY(mesh->NURBSext, "Patchwise integration requires a "
+                        << "NURBS mesh");
```

Comment:
I suspect the patch integration also requires a NURBS FE space -- we can check for that too -- check that `fes->GetNURBSext() != nullptr`:
```suggestion
            MFEM_VERIFY(mesh->NURBSext, "Patchwise integration requires a "
                        << "NURBS mesh");
            MFEM_VERIFY(fes->GetNURBSext(), "Patchwise integration requires a "
                        << "NURBS FE space");
```